CIDER/cider.py

```diff
@@ -60,6 +61,7 @@
 from fpdf import XPos
 from fpdf.enums import Align
 from datetime import date
+import datetime
```

Comment:
You have one specific import from datetime (date) here and then you import the complete package in the next line. Please, either remove the specific import in the first line or turn the second import also into a specific one (preferred option).

Reply:
Ok, I changed that.